app/services/katello/rhsm_fact_parser.rb

```diff
@@ -117,6 +117,10 @@ def kernel_version
       facts['uname.release']
     end
 
+    def convert2rhel
+      facts['conversions.env.CONVERT2RHEL_THROUGH_FOREMAN'].to_i
```

Comment:
I don't know this fact that well, but what does this actually mean? The fact itself implies it's more like a boolean.

I can see it's set here:

https://github.com/theforeman/foreman_ansible/blob/afd7faa412259f339605e5d419a039a72fb30e3b/app/views/foreman_ansible/job_templates/convert_to_rhel.erb#L30-L31

Then in convert2rhel this gets picked up:
https://github.com/oamg/convert2rhel/blob/d738c1dae0a50e0d0c5a7aad60316505fbf70e36/convert2rhel/breadcrumbs.py#L70-L71
https://github.com/oamg/convert2rhel/blob/d738c1dae0a50e0d0c5a7aad60316505fbf70e36/convert2rhel/breadcrumbs.py#L140-L150

So isn't this better?
```suggestion
    def convert2rhel_though_foreman?
      facts['conversions.env.CONVERT2RHEL_THROUGH_FOREMAN'] == "1"
```